reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -56,34 +57,40 @@ export function renderLicences (
 }
 
 function renderLicensesJson (licensePackages: readonly LicensePackage[]) {
-  const data = [
-    ...licensePackages.map((licensePkg) => {
-      return {
-        name: licensePkg.name,
-        version: licensePkg.version,
-        path: licensePkg.path,
-        license: licensePkg.license,
-        licenseContents: licensePkg.licenseContents,
-        author: licensePkg.author,
-        homepage: licensePkg.homepage,
-        description: licensePkg.description,
-      } as LicensePackageJson
-    }),
-  ].flat()
-
-  // Group the package by license
-  const groupByLicense = groupBy((item: LicensePackageJson) => item.license)
-  const groupedByLicense = groupByLicense(data)
-
-  return JSON.stringify(groupedByLicense, null, 2)
+  const data = licensePackages
+    .map((item) => pick(['name', 'version', 'path', 'license', 'author', 'homepage', 'description'], item))
+
+  const output: Record<string, LicensePackageJson[]> = {}
+  const groupedByLicense = groupBy((item) => item.license, data)
+  for (const license in groupedByLicense) {
+    const outputList: LicensePackageJson[] = []
+    const groupedByName = groupBy((item) => item.name, groupedByLicense[license])
+    for (const inputList of Object.values(groupedByName)) {
+      inputList.sort((a, b) => semver.compare(a.version, b.version))
+      const versions = inputList.map((item) => item.version)
+      const paths = inputList.map((item) => item.path ?? null)
```

Comment:
```suggestion
      const paths = inputList.filter((item) => item.path)
```

Reply:
This array corresponds to the versions field. So filter won't be possible.